KsApi/fragments/PPOProjectFragment.graphql

```diff
@@ -0,0 +1,16 @@
+# Minimal version of ProjectFragment, containing only the fields needed for PPO.
+
+fragment PPOProjectFragment on Project {
+  creator {
+    email
+    id
+    name
+  }
+  image {
+    id
+    url(width: 1024)
```

Comment:
Interesting! How does this work?

Reply:
Yeah! I think hardcoded parameters pretty much work the same as the ones we pass in. This one is just copied from our normal project fragment, though